src/common/components/motebehov/MotebehovKvittering.tsx

```diff
@@ -17,10 +18,12 @@ interface Props {
 }
 
 const MotebehovKvittering = ({ motebehov }: Props) => {
+  const { audience } = useAudience();
+
   const behandlerVaereMedTekst =
-    motebehov.skjemaType === "MELD_BEHOV"
-      ? MeldBehovTextsSM.behandlerVaereMedTekst
-      : MeldBehovTextsAG.behandlerVaereMedTekst;
+    audience === "Arbeidsgiver"
```

Comment:
Kan vurdere å bruke isAudienceSykmeldt variabelen i stedet for testlikhet. Kan i teorien få skjulte feil dersom f.eks vi endrer navn fra "Arbeidsgiver" til "Bedrift" eller "Leder" eller noe.

Reply:
Endret